src/data_warehouse/cli/transforms.py

```
@@ -1,97 +1,122 @@
-    if model:
-        click.echo(f"Running transformation for model: {model}")
-        cmd = f"dbt run --select {model}"
-    else:
-        click.echo("Running all transformations")
-        cmd = "dbt run"
-
+@transforms.command()
+@click.option('--models', '-m', help='Specific models to run')
+@click.option('--exclude', '-e', help='Models to exclude')
+@click.option('--profile-dir', help='DBT profile directory', default=os.environ.get('DBT_PROFILES_DIR'))
+@click.option('--project-dir', help='DBT project directory', 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)